docs/advisories/cve_2017_14491.md

```diff
@@ -0,0 +1,48 @@
+# CVE-2017-14491
+
+A vulnerability in dnsmasq, used by kube-dns, requires an upgrade to the kube-dns component.
+
+[More details](https://security.googleblog.com/2017/10/behind-masq-yet-more-dns-and-dhcp.html)
+
+## Fixed in kops versions:
+
+Not yet released
```

Comment:
May want to link to the PR for future context: https://github.com/kubernetes/kops/pull/3511